src/main/resources/db/migration/R__grant_to_cloudsqliamuser.sql

```diff
@@ -1,3 +1,3 @@
--- REVOKE ALL ON ALL TABLES IN SCHEMA public FROM cloudsqliamuser;
--- GRANT SELECT ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
-GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
+REVOKE ALL ON ALL TABLES IN SCHEMA public FROM cloudsqliamuser;
+GRANT SELECT ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
+-- GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
```

Comment:
Hvorfor skru dette av og på flere ganger i master?

Reply:
Vi trengte tilgang til å endre i forbindelse med testingen av fysisk brev, men nå er vi ferdige i denne omgang så da kan vi skru det av igjen. 

Men enig i at vi neste gang kan gjøre dette uten å merge inn i master dersom vi har har bruk for update-tilgang bare i dev-gcp.

Reply:
Ja, det greieste er jo bare å deploye om man vil ha noen midlertidig